### PR TITLE
Refactor README.md

### PR DESCRIPTION
**Pull request by camdenmoors**
Makes the README more clear, and adds deployment/installation instructions for Debian-based distros.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,9 +147,71 @@ This will fetch the latest version of the container, redeploy if a newer version
...
-`docker-compose down` # From the source directory you started from
+From the source directory you started from run:
+
+```bash
+docker-compose down
+```
+
+---
+
+### Heimdall Server - Native
```
Comment (rbclark) [inline README.md:158]: This should probably include some indication that this is not the recommended way to run the application, something like `(Advanced)` next to the heading.

File: README.md [hunk 2]
```diff
@@ -4,128 +4,141 @@
...
-| InSpec Run 'Delta' View                                                        |                                                          | x                                                                             |
-| Multi-Report Tagging, Filtering and Delta View                                 |                                                          | x                                                                             |
+|                                                              | [Heimdall-Lite](https://github.com/mitre/heimdall-lite/) |        [Heimdall](https://github.com/mitre/heimdall/)        |
+| :----------------------------------------------------------- | :------------------------------------------------------: | :----------------------------------------------------------: |
+| Installation Requirements                                    |                      any web server                      |           rails 5.x Server <br /> Postgres Server            |
```
Comment (rbclark) [inline README.md:39]: It is no longer a rails server

File: README.md [hunk 3]
```diff
@@ -4,128 +4,141 @@
...
 3. Navigate to the base folder where `docker-compose.yml` is located
-4. Run the following commands in a terminal window from the heimdall source directory:
-   1. `./setup-docker-secrets.sh`
-   2. `docker-compose up -d`
-6. Navigate to `http://127.0.0.1:3000`
+
+4. Run the following commands in a terminal window from the Heimdall source directory:
+   1. ```bash
+      ./setup-docker-secrets.sh
+      ```
```
Comment (rbclark) [inline README.md:118]: Triple backticks display rather poorly when in a single item list:

Screen Shot 2021-01-12 at 1 52 35 PM

It really only works well on code blocks with 2 or more lines in them.

**Review by wdower — COMMENTED** [review]
Need the one-time postgres config steps up in the install section:

```
cp apps/backend/.env-example apps/backend/.env
# Edit /apps/backend/.env to reflect the appropriate configuration for your system
yarn backend sequelize-cli db:create
yarn backend sequelize-cli db:migrate
yarn backend sequelize-cli db:seed:all
```

Can we clarify a bit on what the difference is between the basic install instructions and the instructions in the "For Developers" section?

**Comment by rbclark** [comment]
> Need the one-time postgres config steps up in the install section:
> 
> ```
> cp apps/backend/.env-example apps/backend/.env
> # Edit /apps/backend/.env to reflect the appropriate configuration for your system
> yarn backend sequelize-cli db:create
> yarn backend sequelize-cli db:migrate
> yarn backend sequelize-cli db:seed:all
> ```
> 
> Can we clarify a bit on what the difference is between the basic install instructions and the instructions in the "For Developers" section?

The issue is we aren't packaging the application in a meaningful installer, we are basically making users build it and run it, so the instructions are very similar. I'm not sure where the requirement came from for manual installation instructions but I'd much rather look at some sort of npx packaging or something similar for Heimdall Server instead of providing manual instructions like this.

**Review by wdower — APPROVED** [review]

**Review by rbclark — APPROVED** [review]

**Pull request merged**